mesheryctl/internal/cli/root/system/stop.go

```diff
@@ -141,6 +144,75 @@ func stop() error {
 			return err
 		}
 
+		//getting service endpoint
+		var opts meshkitkube.ServiceOptions
+		opts.Name = "meshery"
+		opts.Namespace = utils.MesheryNamespace
+
+		clientset := client.KubeClient
+		deploymentsClient := clientset.AppsV1().Deployments(utils.MesheryNamespace)
+
+		endpoint, err := meshkitkube.GetServiceEndpoint(context.TODO(), clientset, &opts)
+		if err != nil {
+			return err
+		}
+		_ = endpoint //temporary line
+
+		//delete deployment
```

Comment:
@navendu-pottekkat use manifests which were used for deployment. https://github.com/layer5io/meshery-operator/blob/master/config/manifests/default.yaml

k8s client shudnt be a part of mesheryctl.

Reply:
@metonymic-smokey Will you use this with a delete option to delete resources? - https://github.com/layer5io/meshkit/blob/master/utils/kubernetes/apply-manifest.go#L32

You wouldn't need any of the client in the code when u use this